pkgs/top-level/all-packages.nix

```diff
@@ -12101,11 +12101,11 @@ with pkgs;
 
   pinentry = libsForQt5.callPackage ../tools/security/pinentry { };
```

Comment:
This will still bring in all the flavours (through `gnupg`) so the overrides below are pointless.

Reply:
That depends on GUI support which by default is only enabled on darwin.

Reply:
Oh, you are right. Looks like this changed two years ago (1ee8f771320b45651d21e4cdbf15141e1c143887). I wonder if it even works without agent then.

Reply:
Lets discover that in another PR to keep the scope manageable.